### PR TITLE
2019 05 07 force secure notification

### PR DESCRIPTION
**Pull request by kermit4**
I haven't tested this, as it requires a lot of dependencies to build.

Please test.

### CODE REVIEW

File: Paubox_Csharp/EmailLib/CommonClasses.cs [hunk 1]
```diff
@@ -28,6 +28,7 @@ public class Message
         public string[] Bcc { get; set; }
         public Header Header { get; set; }
         public bool AllowNonTLS { get; set; } = false;
+        public bool forceSecureNotification { get; set; } = false;
```
Comment (vighneshtrivedi2004) [inline Paubox_Csharp/EmailLib/CommonClasses.cs]: Please change the property name to ForceSecureNotification.

File: Paubox_Csharp/EmailLib/packages.config [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net461" />
-</packages>
\ No newline at end of file
+  <package id="Newtonsoft.Json" version="10.0.1" targetFramework="net461" />
```
Comment (vighneshtrivedi2004) [inline Paubox_Csharp/EmailLib/packages.config]: This change is not required. Please undo changes in this file

File: Paubox_Csharp/UnitTestProject/SendMessage_TestData.csv [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-7,vighneshtrivedi2004@gmail.com,,Test Mail,renee@undefeatedgames.com,,,Hello Vighnesh,<html><body><h1>Hello World!</h1></body></html>,1,hello_world.txt,fg,SGVsbG8gV29ybGQh\n,SUCCESS
-8,,,Test Mail,renee@undefeatedgames.com,,,Hello Vighnesh,,0,,,,ERROR
-9,vighneshtrivedi2004@gmail.com,,,renee@undefeatedgames.com,,,Hello Vighnesh,,0,,,,ERROR
-10,vighneshtrivedi2004@gmail.com,,Test Mail,,,,Hello Vighnesh,,0,,,,ERROR
-11,vighneshtrivedi2004@gmail.com,,Test Mail 2,renee@undefeatedgames.com,,,,,0,,,,SUCCESS
-12,vighneshtrivedi2004@gmail.com,,,renee@undefeatedgames.com,,,,,0,,,,ERROR
-13,vighneshtrivedi2004@gmail.com,,,,,,Hello Vighnesh,,0,,,,ERROR
-14,,,Test Mail,,,,Hello Vighnesh,,0,,,,ERROR
-15,,,,,,,,,0,,,,ERROR
+S.No,Recipient,Bcc,Subject,From,Reply_To,AllowNonTLS ,Text_Plain,Text_Html,Attachments,FileName,ContentType,Content,,forceSecureNotification,Expected Response
```
Comment (vighneshtrivedi2004) [inline Paubox_Csharp/UnitTestProject/SendMessage_TestData.csv]: Please change the column name to ForceSecureNotification.

File: Paubox_Csharp/UnitTestProject/SendMessage_TestData.csv [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+1,vighneshtrivedi2004@gmail.com,vighneshtrivedi2004@gmail.com,Test Mail,renee@undefeatedgames.com,renee@undefeatedgames.com,TRUE,Hello Vighnesh,<html><body><h1>Hello World!</h1></body></html>,1,hello_world.txt,text/plain,SGVsbG8gV29ybGQh\n,,SUCCESS
```
Comment (vighneshtrivedi2004) [inline Paubox_Csharp/UnitTestProject/SendMessage_TestData.csv]: Add TRUE or FALSE value for ForceSecureNotification column, for any 1 Success and 1 Error Scenario.

**Comment by kermit4** [comment]
I have made the changes.

Please apply any further changes you find needed.

**Pull request merged**